### DISCUSSION TITLE
Casper 2.0 Local Development

### DISCUSSION THREAD

**Discussion by melpadden** [casper-network/condor-info#4] [Articles]
Is it possible to run Condor locally using CCTL or NCTL? Has this been tested?

**Comment by ghost** [comment]
Re CCTL support: Casper 2.0 is currently supported upon the [dev](https://github.com/casper-network/cctl/tree/dev) branch.

**Comment by melpadden** [comment]
[This](https://github.com/casper-network/condor-info/blob/main/articles/001.md) article explains changes in the node setup.

**Comment by mssteuer** [comment]
NCTL can be used with these instructions: https://github.com/make-software/casper-nctl-docker/blob/feat-2.0/BUILD%20Condor.md
